web/components/chat.css

```diff
@@ -6,6 +6,18 @@
  * https://opensource.org/licenses/BSD-3-Clause
  */
 
+.file-dragged-over {
+  border-bottom: 2px solid #005fb2;
+  border-left: 2px solid #005fb2;
```

Comment:
should we extract these colours into the colours file ?

Reply:
@BrandonMurnane maybe I'm misunderstanding something here but I don't think we can read values from a `.js` file into a `.css` file